packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -84,6 +84,7 @@ export class Calendar extends React.Component<ICalendarProps, ICalendarState> im
 
   private _dayPicker = React.createRef<ICalendarDay>();
   private _monthPicker = React.createRef<ICalendarMonth>();
+  private isFocusInside = false;
```

Comment:
```suggestion
  private _hasFocus = false;
```
1. pls add `_` prefix for private field
2. i think hasFocus is a simpler name that better described its use?